Sources/Xcodeproj/pbxproj.swift

```diff
@@ -73,8 +73,10 @@ public func xcodeProject(
         guard let manifestLoader = options.manifestLoader else { return }
 
         let pdTarget = project.addTarget(
-            objectID: "\(package.manifestName)::SwiftPMPackageDescription", // TODO: use identity instead?
-            productType: .framework, name: "\(package.manifestName)PackageDescription") // TODO: use identity instead?
+            objectID: "\(package.manifest.displayName)::SwiftPMPackageDescription", // TODO: use identity instead?
+            productType: .framework,
+            name: "\(package.manifest.displayName)PackageDescription" // TODO: use identity instead?
+        )
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
Probably we should use the display name here for the name but an identifier for the object ID.